packages/ui-react/src/components/FormField/FormField.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import useOpaqueId from '@jwp/ott-hooks-react/src/useOpaqueId';
+
+import HelperText from '../HelperText/HelperText';
+import type { FieldProps } from '../../types/form-fields';
+
+import styles from './FormField.module.scss';
+
+type Props = {
+  children: (inputProps: { helperTextId: string; id: string }) => React.ReactNode;
+  showOptionalLabel?: boolean;
+  helperText?: React.ReactNode;
```

Comment:
Should it be part of FieldProps?

Reply:
FieldProps is used for control and field component props. Perhaps I should rename FieldProps to FormControlProps, because `helperText` is for form fields only. What do you think?